projects/packages/my-jetpack/_inc/data/products/use-install-standalone-plugin.ts

```diff
@@ -3,8 +3,22 @@ import { REST_API_SITE_PRODUCTS_ENDPOINT } from '../constants';
 import { QUERY_INSTALL_PRODUCT_KEY } from '../constants';
 import useSimpleMutation from '../use-simple-mutation';
 import useProduct from './use-product';
+import type { APIFetchOptionsWithQueryParams } from '../use-simple-mutation';
+import type { UseMutateFunction } from '@tanstack/react-query';
 
-const useInstallStandalonePlugin = ( productId: string ) => {
+export type InstallAction = UseMutateFunction<
```

Comment:
It's just nomenclature, so don't hesitate to ignore that 😅 This time seems to me more like "Callback" or "Function", rather than "Action".